src/Lykke.Job.TransactionHandler/Modules/CqrsModule.cs

```diff
@@ -247,18 +237,6 @@ protected override void Load(ContainerBuilder builder)
                         .From(BoundedContexts.Solarcoin).On(defaultRoute)
                     .WithProjection(typeof(EmailProjection), BoundedContexts.Solarcoin),
 
-                Register.Saga<CashInOutSaga>($"{BoundedContexts.TxHandler}.cash-out-saga")
-                    .ListeningEvents(typeof(CashoutTransactionStateSavedEvent))
-                        .From(BoundedContexts.Operations).On(defaultRoute)
-                    .PublishingCommands(typeof(BitcoinCashOutCommand))
-                        .To(BoundedContexts.Bitcoin).With(defaultPipeline)
-                    .PublishingCommands(typeof(ProcessEthereumCashoutCommand))
-                        .To(BoundedContexts.Ethereum).With(defaultPipeline)
-                    .PublishingCommands(typeof(SolarCashOutCommand))
-                        .To(BoundedContexts.Solarcoin).With(defaultPipeline)
-                    .PublishingCommands(typeof(BlockchainCashoutProcessor.Contract.Commands.StartCashoutCommand))
```

Comment:
StartCashoutCommand class is not deleted. Is it ok?

Reply:
What do you mean?

Reply:
Sorry, I thought it is our class.